examples/basic/basic.c

```diff
@@ -51,7 +51,7 @@ int main(void) {
 
     // Write to the allocated memory
     memset(ptr_provider, '\0', alloc_size);
-    util_strncpy(ptr_provider, alloc_size, strSource, strlen(strSource) + 1);
+    strncpy(ptr_provider, strSource, alloc_size);
```

Comment:
Please remove '#include "utils_common.h"' from this file. It's no longer needed after this change.